composer.json

```diff
@@ -20,20 +20,22 @@
             "homepage": "https://github.com/friendsofsymfony/FOSHttpCache/contributors"
         }
     ],
+    "minimum-stability": "dev",
```

Comment:
> For the time being php http packages are mostly unstable as well,
> This minimum-stability is necessary because some of the php-http packages themselves require unstable versions of other php-http packages

Any chance those will be stable in the near future *(by the time 2.0 of FOSHttpCache is stable)*?
Otherwise might have to document which packages are unstable in order for people to specify every single one in project install to avoid having to set `minimum-stability` to dev.